src/content/en/updates/2017/03/background_tabs.md

```diff
@@ -0,0 +1,102 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Background tabs in Chrome 57. 
+
+{# wf_updated_on: 2017-03-09 #}
+{# wf_published_on: 2017-03-09 #}
+{# wf_tags: chrome57 #}
+{# wf_featured_snippet: Background tab behavior in Chrome 57. #}
```

Comment:
This is your hook and is used on the index pages as the description for the article, and should be more descriptive than this. Think along the lines "To improve performance, make you a better person, create world peace, starting in chrome 57, the behaviour of background tabs is changing. What action do devs need to take or be aware of."